src/Integration/WordPressEmails.php

```diff
@@ -145,12 +145,26 @@ public function disable_send_confirmation_on_admin_email() {
 
 	}
 
+	/**
+	 * Disables email on admin email address changed
+	 *
+	 * @action notification/init
+	 *
+	 * @since 8.0.15
+	 * @return void
+	 */
+	public function disable_send_confirmation_on_admin_email_changed() {
+		if ( 'true' === notification_get_setting( 'integration/emails/send_confirmation_on_admin_email_changed' ) ) {
+			add_filter( 'send_site_admin_email_change_email', '__return_false' );
+		}
+	}
```

Comment:
Wrong coding standards, and the wrong branch. It should be branched from `major`, PR should be back to `major` as well